<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carRecord</class>
 <widget class="QWidget" name="carRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>521</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>车牌号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>进入时间         </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>离开时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>车辆类型</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>419</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>请输入</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>车牌号</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>进入时间(XX-YY-ZZ AA:BB:CC)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>离开时间(XX-YY-ZZ AA:BB:CC)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>内部车</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>外部车</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
